lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py

```diff
@@ -217,7 +217,7 @@ def main():
     argument_spec = cs_argument_spec()
     argument_spec.update(dict(
         name=dict(required=True),
-        affinty_type=dict(),
+        affinity_type=dict(aliases=['affinty_type']),
```

Comment:
Lets deprecate the "old" param: 
~~~diff
-        affinty_type=dict(),
+        affinty_type=dict(removed_in_version='2.9'),
+        affinity_type=dict(),